module-1/lab-data_cleaning/your-code/.ipynb_checkpoints/weather-checkpoint.ipynb

```diff
@@ -11,31 +11,1196 @@
   },
```

Comment:
Good try!! But that's not the direction... If you have interest on doing it on your own, we can seat 5-10 minutes in any slot.